source/fundamentals/data-formats/document-data-format-bson.txt

```diff
@@ -38,7 +38,7 @@ MongoDB and BSON
 
 The MongoDB Java Driver, which uses the BSON library, allows you to work
 with BSON data by using one of the object types that implements the
-:java-docs:`Bson <apidocs/bson/org/bson/conversions/Bson.html>` interface,
+:java-docs:`BSON <apidocs/bson/org/bson/conversions/Bson.html>` interface,
```

Comment:
I think these links are the same, but shouldn't the linked text be the same, at least in this instance?
https://github.com/mongodb/docs-java/pull/146/files#diff-b4955d68d249252f5472f742c01958cda1df25f3541a79865dd4d3e61314e836R57